pynitrokey/start/usb_strings.py

```diff
@@ -42,6 +41,7 @@ def get_dict_for_device(dev: usb.Device) -> dict:
 
 
 def get_devices() -> list:
+    from pynitrokey.start.gnuk_token import gnuk_devices_by_vidpid
```

Comment:
`gnuk_devices_by_vidpid` should be extracted to a separate module

Reply:
next PR